### DISCUSSION TITLE
Picture in Center

### DISCUSSION THREAD

**Discussion by daprogrammer2022** [Shane32/QRCoder#424] [Q&A]
Is there a way to put a picture in the center of a QR Code coder?

**Comment by MattOpen** [comment]
Yes it is possible

**Reply by nzhuravel** [reply]
Documentation says the image overload is present in Base64 renderer:
https://github.com/codebude/QRCoder/wiki/Advanced-usage---QR-Code-renderers#23-base64qrcode-renderer-in-detail
But in reality (v1.6.0) it is not there. Is it a typo in docs?

**Reply by nzhuravel** [reply]
aha, I think I got it, the code source has a check for .Net version. I am running .net8, so the necessary functionality is disabled. Sad.
https://github.com/codebude/QRCoder/blob/master/QRCoder/Base64QRCode.cs#L121

**Reply by codebude** [reply]
It's available also for .net8, but only under Windows environment as it needs System.Drawing for the logo rendering.

**Comment by mahdiit** [comment]
you can load svg (it is best better than rasterized image) or if you want image (using cross prlatform loader) in my repo
see sample, Gufel.QrRender based on qrcoder
https://github.com/mahdiit/Gufel.QrRender/blob/master/Asset/template.md
